README.md

```diff
@@ -44,16 +49,28 @@ gotk check --pre
 
 ## Get started
 
-To get started with the GitOps Toolkit, start [browsing the documentation](https://toolkit.fluxcd.io)
-or get started with one of the following guides:
+To get started with Flux, start [browsing the
+documentation](https://toolkit.fluxcd.io) or get started with one of
+the following guides:
 
 - [Get started with GitOps Toolkit (deep dive)](https://toolkit.fluxcd.io/get-started/)
 - [Installation](https://toolkit.fluxcd.io/guides/installation/)
 - [Manage Helm Releases](https://toolkit.fluxcd.io/guides/helmreleases/)
 - [Setup Notifications](https://toolkit.fluxcd.io/guides/notifications/)
 - [Setup Webhook Receivers](https://toolkit.fluxcd.io/guides/webhook-receivers/)
 
-## Components
+## <a name="gitops-toolkit">GitOps Toolkit</a>
```

Comment:
I think GitHub does this automatically? See, e.g: https://github.com/fluxcd/toolkit/#Components

Reply:
Oh yes, I forgot about that.